src/DashboardUI/src/pages/application/estimation-tool/EstimationToolSidebar.tsx

```diff
@@ -1,14 +1,174 @@
-import MapTheme from '../../../components/home-page/water-rights-tab/map-options/components/MapTheme';
-import { NotImplementedPlaceholder } from '../../../components/NotImplementedAlert';
+import { MapThemeSelector } from '../../../components/map/MapThemeSelector';
+import { OverlayTooltip } from '../../../components/OverlayTooltip';
+import Icon from '@mdi/react';
+import { mdiPiggyBank, mdiWater } from '@mdi/js';
+import { Form, InputGroup } from 'react-bootstrap';
+import {
+  CompensationRateUnitsLabels,
+  CompensationRateUnitsOptions,
+} from '../../../data-contracts/CompensationRateUnits';
+import { FundingOrganizationDetails } from '../../../data-contracts/FundingOrganizationDetails';
+
+interface EstimationToolSidebarProps {
+  fundingOrganizationDetails: FundingOrganizationDetails | undefined;
+  isLoadingFundingOrganization: boolean;
+}
+
+export function EstimationToolSidebar(props: EstimationToolSidebarProps) {
+  const acreageSum = 4681;
+  const evapotranspiration = 1000;
+  const conservationEstimate = 15000;
 
-export function EstimationToolSidebar() {
   return (
     <div className="flex-grow-1 panel-content">
       <div className="container-fluid">
-        <NotImplementedPlaceholder />
+        <SidebarElement title="CALCULATED SHAPE AREA FOR ALL IRRIGATED FIELDS">
```

Comment:
I wonder if we should use style to uppercase transform this, instead of the values themselves. I typically use css since it's easier to update if it changes

```suggestion
        <SidebarElement title="Calculated Shape Area for all Irrigated Fields">
```

Reply:
that was actually my first thought, but the OpenET header has lowercase characters in it, so I think it makes more sense currently to have the text just be exactly what it should be rather than trying to conditionally transform it.

![image](https://github.com/user-attachments/assets/00daaf70-33d2-4e92-acee-4056ec95621e)